composer.json

```diff
@@ -18,7 +18,8 @@
         "symfony/framework-bundle": "5.1.*",
         "symfony/twig-bundle": "5.1.*",
         "symfony/yaml": "5.1.*",
-        "twig/intl-extra": "^3.1"
+        "twig/intl-extra": "^3.1",
+        "ext-json": "*"
```

Comment:
why?

Reply:
The project has `JsonSerializable` usages and it's not as default there.